### PR TITLE
Feature/sidepanel resize

### PR DESCRIPTION
**Pull request by jakublamprecht**
## Description

Added possibility of resizing sidepanel and updated sidepanel toggle functionality to work correctly with different widths of sidepanel

## Motivation and Context

Test names can get pretty long and are hard to read with current width of the sidepanel.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](https://github.com/Cognifide/aet/blob/master/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: report/src/main/webapp/app/layout/sidepanel/sidepanelResizer.directive.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      },
+      link: linkFunc
+    };
+
+    function linkFunc($scope, $element) {
+      var $body = $('body');
+      var $content = $('.main');
+      var $sidepanel = $('.aside');
+
+      var isSidepanelResized = false;
```
Comment (mateuszluczak) [inline report/src/main/webapp/app/layout/sidepanel/sidepanelResizer.directive.js]: Move consts outside link function

File: report/src/main/webapp/app/layout/sidepanel/sidepanelResizer.directive.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      restrict: 'AE',
+      scope: {
+        'type': '@'
+      },
+      link: linkFunc
+    };
+
+    function linkFunc($scope, $element) {
+      var $body = $('body');
+      var $content = $('.main');
```
Comment (mateuszluczak) [inline report/src/main/webapp/app/layout/sidepanel/sidepanelResizer.directive.js]: Is there a better way of getting that elements? Can't the directive be attached to higher element in DOM and use elements in it's scope?

File: report/src/main/webapp/app/layout/sidepanel/sidepanelToggle.directive.js [hunk 3]
```diff
@@ -22,6 +22,8 @@ define(['angularAMD'], function (angularAMD) {
       ['$rootScope', 'localStorageService', ToggleSidepanelDirective]);
 
   function ToggleSidepanelDirective($rootScope, localStorageService) {
+    var $content = $('.main');
```
Comment (mateuszluczak) [inline report/src/main/webapp/app/layout/sidepanel/sidepanelToggle.directive.js]: This is also duplicated in the second directive, there should be a way of making that in single place

File: report/src/main/webapp/app/layout/sidepanel/sidepanel.directive.js [hunk 4]
```diff
@@ -94,32 +91,38 @@ define(['angularAMD'], function (angularAMD) {
...
+      updateWidth(limitSidepanelSize($sidepanel.outerWidth()));
+    }
+
     function expand() {
       $content.css('left', $sidepanel.outerWidth());
-      $content.css('width', $body.width() - $sidepanel.outerWidth());
+      $content.css('width', document.body.clientWidth - $sidepanel.outerWidth());
       $sidepanel.css('left', 0);
 
+      $(window).on('resize', onWindowResize);
```
Comment (mateuszluczak) [inline report/src/main/webapp/app/layout/sidepanel/sidepanel.directive.js]: Add throttle here

File: report/src/main/webapp/app/layout/sidepanel/sidepanel.directive.js [hunk 5]
```diff
@@ -94,32 +91,38 @@ define(['angularAMD'], function (angularAMD) {
...
       localStorageService.put(EXPANDED_SIDEBAR_KEY_NAME, true);
     }
 
     function close() {
       $content.css('left', 0);
-      $content.css('width', $body.width());
+      $content.css('width', '100%');
```
Comment (mateuszluczak) [inline report/src/main/webapp/app/layout/sidepanel/sidepanel.directive.js]: Can't we just remove width then?

Comment (jakublamprecht) [reply report/src/main/webapp/app/layout/sidepanel/sidepanel.directive.js]: Yup, removed width inline style instead

File: report/src/main/webapp/app/layout/sidepanel/sidepanel.directive.js (hunk 4 above)
Comment (jakublamprecht) [reply report/src/main/webapp/app/layout/sidepanel/sidepanel.directive.js]: Done

**Pull request merged**